overlord/snapstate/snapstate.go

```diff
@@ -1909,6 +1933,22 @@ func Remove(st *state.State, name string, revision snap.Revision, flags *RemoveF
 		if tp, _ := snapst.Type(); tp == snap.TypeApp && removeAll {
 			ts, err := AutomaticSnapshot(st, name)
 			if err == nil {
+				sz, err := EstimateSnapshotSize(st, name, nil)
+				if err != nil {
+					return nil, err
+				}
+				// require 5Mb extra
+				requiredSpace := sz + 5*1024*1024
+				path := dirs.SnapdStateDir(dirs.GlobalRootDir)
```

Comment:
Ah so path is a fixed location. In that case that's probably fine. You can ignore my comment above.